paasta_tools/cli/cmds/spark_run.py

```diff
@@ -473,6 +506,10 @@ def _parse_user_spark_args(spark_args: Optional[str]) -> Dict[str, str]:
             )
             sys.exit(1)
         user_spark_opts[fields[0]] = fields[1]
+
+    if enable_k8s_autogen:
+        user_spark_opts["spark.kubernetes.executor.podTemplateFile"] = POD_TEMPLATE_PATH
```

Comment:
Make `podTemplateFile` name unique. It doesn't matter if we don't delete the file afterwards, it's only a few bytes. @nemacysts do you know how often `/nail/tmp` gets cleared on prod batch boxes?

Reply:
not off the top of my head - i can check what tmpwatches we have setup on them in puppet tho

Reply:
Yeah I think i'll make the file name something like spark_pt_{uid}, and if we ever want to clean up after we can always set up a seperate cronjob for it on batch box

Reply:
whoops sorry - forgot to respond: we do have a generic tmpwatch that runs daily, but it will only clean things up that are older than 10 days